docs/doc/code-review.md

```diff
@@ -3,15 +3,22 @@ id: code-review
 title: How to Conduct A Helpful Code Review
 ---
 
-Consider following links:
+As a reviewer consider the following guidelines:
 
 - [How to Do Code Reviews Like a Human (Part One)](https://mtlynch.io/human-code-reviews-1/) /
   [Code review по-человечески (часть 1)](https://habr.com/en/post/340550/)
 - [How to Do Code Reviews Like a Human (Part Two)](https://mtlynch.io/human-code-reviews-2/) /
   [Code review по-человечески (часть 2)](https://habr.com/ru/post/342244/)
 - [PR Code Review Phrase Book](https://gist.github.com/OleksiyRudenko/e6f573d7aca2cc854ccce6087cfe7138)
 
-## Formatting, Style, Manual Testing
+## General requirements
+
+The code is written for other people to read and contribute to comfortably.
```

Comment:
Suggesting to add a preposition and an excplicit reference to the object of contribution, for the sake of clarity.
```suggestion
The code is written for other people to read and to contribute to the code comfortably.
```